src/main/java/com/lootingbag/GeUntradables.java

```diff
@@ -0,0 +1,12 @@
+package com.lootingbag;
+
+public class GeUntradables
+{
+	/**
+	 * All ItemIDs for items that are tradeable but cannot be bought/sold on the Grand Exchange
+	 * Scraped from <a href="https://oldschool.runescape.wiki/w/Grand_Exchange/Non-tradeable_items"/>
+	 */
+	public static final int[] ITEM_IDS = new int[] {
```

Comment:
Please use the runelite ItemID API to avoid hardcoded IDs. Also this would be better as a HashSet since you just do contains calls on it in constant time.